packages/jsii-python-runtime/tests/test_compliance.py

```diff
@@ -1073,3 +1075,15 @@ def test_can_use_interface_setters():
 def test_structs_are_undecorated_on_the_way_to_kernel():
     json = JsonFormatter.stringify(StructB(required_string='Bazinga!', optional_boolean=False))
     assert loads(json) == {'requiredString': 'Bazinga!', 'optionalBoolean': False}
+
+def test_pure_interfaces_can_be_used_transparently():
+    expected = StructB(required_string="It's Britney b**ch!")
+
+    @jsii.implements(IStructReturningDelegate)
```

Comment:
Do we still need this `@jsii.implements`??

Reply:
Yes we do.

Reply:
`@property` declarations are idiomatic as far as I know, but the @jsii stuff is not.